Sources/Logging/DiagnosticsLogger.swift

```diff
@@ -163,7 +163,7 @@ extension DiagnosticsLogger {
                     self?.logSize += Int64(data.count)
                     self?.trimLinesIfNecessary()
                 } catch {
-                    assertionFailure("Writing data failed with error: \(error)")
+                    print("Writing data failed with error: \(error)")
```

Comment:
What was the reason behind this?

Reply:
Tests were failing locally due to the assertion failure